documentation/src/main/asciidoc/server_guide/server_guide.adoc

```diff
@@ -1007,3 +1046,38 @@ remoteCacheManager = new RemoteCacheManager(clientBuilder.build());
 
 Additionally, you might also want to enable client certificate authentication (and therefore also allow the use of the EXTERNAL SASL mech to authenticate clients).
 
+
+
+== Multi-tenancy
+
+Multi-tenancy allows accessing multiple containers as shown below:
+
+image::images/multi-tenancy.png.png[]
+
+Currently there are two supported protocols for accessing the data - using Hot Rod client and using REST interface.
+
+=== Using REST interface
+
+Multi-tenancy router uses URL prefixes to separate containers using the following scheme: `http://<server_ip>:<server_port>/rest/<rest_connector_name>/<cache_name>/<key>`. All HTTP operations remain exactly the same as using standard `rest-connector`.
+
+=== Using Hot Rod client
+
+Multi-tenant routing for binary protocols requires using a standard, transport layer mechanism such as link:$$https://en.wikipedia.org/wiki/Server_Name_Indication$$[SSL/TLS Server Name Indication]. The server needs to be configured to support encryption and additional SNI routing needs to be added to the `router-connector`.
```

Comment:
Do we have plans to support something other than SNI? If not, then you can say more directly "requires SSL and SNI".